net/pbr/files/README.md

```diff
@@ -0,0 +1,3 @@
+# README
```

Comment:
Hmm, readme should be here or it should be removed.

Reply:
I strongly prefer to host the README and the screenshots it includes together, which is a no-go for this repo. Currently the README (and the screenshots) are hosted in another github repo and published thru github pages. 

Some people may discover/look for the README within the package directory that's why it's here.